RobotBase.java

```diff
@@ -117,20 +146,18 @@ public static synchronized RobotBase getInstance() {
   public void disabledInit() {
     robotContainer.disabled();
   }
-  /**
-   * This function runs called periodically while the robot is disabled.
-   */
+
   @Override
   public void disabledPeriodic() {
     robotContainer.disabledPeriodic();
   }
 
   /**
-   * This autonomous runs the autonomous command selected by Sebby
+   * This autonomous runs the autonomous command selected by your
    */
   @Override
   public void autonomousInit() {
-    Logger.consoleLog("Autonomous Init");
+    // Logger.consoleLog("Autonomous Init");
```

Comment:
why not just delete these lines rather than commenting them out?

Reply:
I thought I deleted them, i'll get on that tomorrow

Reply:
When you create a PR: look at the diff before you send it out. At least skim through it. You'll find something that you thought you'd done or forgot about or whatever very nearly every time. And if you go through it as if you were the reviewer, you'll probably find quite a few things you want to change before you sent it out. Or maybe it's just me - I know I usually do. It's that self-editing step that your English teachers have been talking about since you learned to write. Same thing applies here.

Reply:
Yeah, I definitely created this PR hastily. Well, its just a lesson for the future.